internal/services/oracle/validate/exascale_database.go

```diff
@@ -23,3 +23,19 @@ func ExascaleDatabaseStorageVaultName(i interface{}, k string) (warnings []strin
 
 	return
 }
+
+func ExascaleDatabaseVirtualMachineClusterName(i interface{}, k string) (warnings []string, errors []error) {
+	v, ok := i.(string)
+	if !ok {
+		return nil, append(errors, fmt.Errorf("expected type of %s to be string", k))
+	}
+
+	// The value must start with a letter or an underscore (_).
+	// The value can only include letters, numbers, underscores (_), and hyphens (-).
+	// The value cannot contain consecutive hyphens (--).
+	if !regexp.MustCompile(`^[a-zA-Z_](?:[a-zA-Z0-9_]*(?:-[a-zA-Z0-9_]+)*-?)?$`).MatchString(v) {
```

Comment:
Is there a need to explicitly use the non-capture group syntax here?

Reply:
I meant to remove the [*non-capture group*](https://pkg.go.dev/regexp/syntax#:~:text=(%3F%3Are)%20%20%20%20%20%20%20%20%20non%2Dcapturing%20group) syntax, which is not necessary here.